.env

```diff
@@ -0,0 +1,3 @@
+GMAIL_USERNAME=tokenowo.mail@gmail.com
+GMAIL_PASSWORD=H7cLSeRwPu6GqtV
+SECRET_KEY_BASE=q7XvOCjOYEpsozZvcIOlGsUFV3rcUF5Eq3MfW4/Hi1O0n3XeErUMmMd5KENYHlFuqJvCmV7bbMQOp0NO25ahhrlfaT5m1lEmSRtqsadaB837bpwhr/NKwcGrUzmuOPr4BP2wwHfqVRg27qS8MvJ6Afmhv1e0rJsyxY0sxFNGWPhrHljob+VB2HxukDjc0et/cExrAhRvdherQ64pikW4YjtqnL4Ni+e3qDShIZhMyb+s6H/+g4Eh0G+mVT90m6TZ8jKhsfjfdFv9AS47f8w/JZwSQL+xJvLrNfp2pIZdPxPdrai9X0bFejFP5DJ+7vk7ZSNTwRmuDXmf57Ok9l/IVaNiV6fquAJRW5a1M4x+qnkFiM+rvyJm3ed+21+C4stOVanhxrAcR841eGWg4tyO0P+bUZcrWM9liaNu--VTgPMFSZjEvr07BP--76zFuPiOIH9oCxxobzPTNw==
```

Comment:
You have added `SECRET_KEY_BASE` here but you don't use it anywhere. Please use it in on those places where you have said you will change during your commits